src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Contracts/Serialization.cs

```diff
@@ -114,43 +114,43 @@ public SerializeDataStartRequestParams()
             this.IsLastBatch = isLast;
         }
 
-        internal bool IncludeHeaders
+        public bool IncludeHeaders
```

Comment:
Huh, so these being internal meant that they weren't being assigned values from the request when being deserialized?

Reply:
Yeah, the request object gets created in Microsoft.SqlTools.Hosting, but the Params object is in ServiceLayer. All the internal fields always showed up as null, even though the corresponding properties had real values in the RPC message.